lua/pixelui/core/cl_color.lua

```diff
@@ -130,4 +130,22 @@ function PIXEL.GetRainbowColor()
     lastCol = HSVToColor((time * 50) % 360, 1, 1)
 
     return lastCol
+end
+
+local colorMeta = FindMetaTable("Color")
+
+function colorMeta:Copy()
+    return PIXEL.CopyColor(self)
+end
+
+function colorMeta:Offset(off)
+    return PIXEL.OffsetColor(self, off)
+end
+
+function colorMeta:Lerp(amt, to)
+    return PIXEL.LerpColor(amt, self, to)
+end
+
+function colorMeta:__eq(to)
```

Comment:
You can't do this and guarantee that there aren't any side effects, so don't.

You could make an Equal/EqualTo method instead, preferably EqualTo.